src/metabase/public_settings.clj

```diff
@@ -262,9 +262,9 @@
   :audit      :getter)
 
 (defsetting enable-query-caching
-  (deferred-tru "Enabling caching will save the results of queries that take a long time to run.")
+  (deferred-tru "Allow caching results of queries that take a long time to run.")
   :type       :boolean
-  :default    false
+  :default    true
```

Comment:
I assume this was agreed by some product person?

Reply:
yeah Arakaki himself 😁